src/common/components/Pagination/index.js

```diff
@@ -14,45 +14,74 @@ import {
   Bold,
 } from "./styled";
 import { useSelector } from "react-redux";
-import { selectMovieList } from "../../../features/movies/MovieList/movieListSlice";
-import { selectPeopleList } from "../../../features/people/PeopleList/peopleListSlice";
+import { selectTotalMoviesPages } from "../../../features/movies/MovieList/movieListSlice";
+import { selectTotalPeoplePages } from "../../../features/people/PeopleList/peopleListSlice";
 import { useLocation } from "react-router-dom";
-import { useSearch } from "../../../features/useSearch";
+import { useSearch, useReplaceQueryParameter } from "../../../features/useSearch";
 import pageQueryParameter from "../../../features/pageQueryParameter";
 
 export const Pagination = () => {
   const location = useLocation();
   const atMovies = location.pathname.includes("movies") ? true : false;
   const currentPage = +useSearch(pageQueryParameter) || 1;
-  const totalMoviesContent = useSelector(selectMovieList);
-  const totalPeopleContent = useSelector(selectPeopleList);
-  const totalPages = atMovies ? totalMoviesContent : totalPeopleContent;
+  const totalMoviesPages = useSelector(selectTotalMoviesPages);
+  const totalPeoplePages = useSelector(selectTotalPeoplePages);
+  const totalPages = atMovies ? totalMoviesPages : totalPeoplePages;
   const inactivePrevious = currentPage === 1;
   const inactiveNext = currentPage === totalPages;
+  const replaceQueryParameter = useReplaceQueryParameter();
+
+  const onClickButtonFirst = () => {
+    replaceQueryParameter({ key: pageQueryParameter, value: 1 })
+  };
+
+  const onClickButtonPrevious = () => {
+    replaceQueryParameter({ key: pageQueryParameter, value: +currentPage - 1 });
```

Comment:
Good idea for writting a variable with '+' sign before.

Reply:
thanks!